USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -483,7 +483,8 @@ private string RenderFootnotes()
             {
                 
                 StringBuilder footnoteHTML = new StringBuilder();
-                footnoteHTML.AppendLine("<div class=\"footnote-header\">Footnotes</div>");
+                footnoteHTML.AppendLine("<hr/>");
+                footnoteHTML.AppendLine("<div class=\"footnote-header\"></div>");

```

Comment:
Why add the extra div here? It isn't making sense in my head.

Reply:
@rbnswartz The div tag contained the word 'Footnotes' for styling. You're right, it's not necessary.